### DISCUSSION TITLE
Working example for Webpack & React with fast refresh?

### DISCUSSION THREAD

**Discussion by fellmann** [swc-project/swc#1697] [Q&A]
I have been playing around with swc, webpack, typescript and react setup. Although according to the documentation swc supports fast refresh for React, I found no working example to make it work with webpack. Has anyone done this yet or knows a working demo?

**Comment by haydnhkim** [comment]
@Austaras Can you give an example of using fast refresh in swc?
I set jsc.transform.react.refresh to true but full reload occurs.
I don't know how to make fast refresh work.

**Reply by Austaras** [reply]
https://github.com/Austaras/ynnari/blob/master/src/webpack.config.ts

**Reply by haydnhkim** [reply]
@Austaras OMG, thanks for the super quick reply!
